lectures/scipy.md

```diff
@@ -63,7 +63,8 @@ from numpy.fft import fft, ifft
 from numpy.lib.scimath import *
 ```
 
-However, it's more common and better practice to use NumPy functionality explicitly
+However, it's more common and better practice to use NumPy functionality explicitly. One good feature of Numpy is it has a faster processing time than other Python libraries.
```

Comment:
@Horace510 : The statement you added is not precise enough.  Sometimes other libraries are faster.  (Also, Numpy should be NumPy.)

Also, the point here is that you can bring in NumPy implicitly, via SciPy, or explicitly.  It's better to be explict.  

How about adding ", rather than implicitly via SciPy" ?